lib/builder.ts

```diff
@@ -28,6 +27,13 @@ interface BuilderOptions {
 
 type NodeWrappers = TransformNodeWrapper | SourceNodeWrapper;
 
+// This can be await'ed to force everything that follows to run in the next tick.
+function forceAsync(): Promise<void> {
```

Comment:
I don't think we need this, `await Promise.resolve()` + comment in the calling code should be sufficient.

Reply:
I thought that code might not be sufficient to force a next tick, but it seems to do the trick.